### ISSUE TITLE
Report preview function not working

### ISSUE THREAD

**Issue by whitingjr** [Hyperfoil/Horreum#399]
**Describe the bug**

A Report should load when the Preview button is clicked.
Instead it is  showing this pop-up message in the client UI

```
Warning alert:Failed to generate report preview.

{
  "details": "Error id 48d3aa5f-dd62-46f4-a745-bfacebf4ef24-25",
  "stack": ""
}
```
The client request sent was processed and responded too

```
XHRPOSThttps://acme.com/api/report/table/preview
[HTTP/1.1 500 Internal Server Error 155ms]
```

The server logs are also not reporting any ERROR/WARN or DEBUG message in the logs. Despite returning to the client browser an SC 500 response code.

**To Reproduce**

Select an existing report that is known to have worked.
 Click the Preview button.

**Version**

0.7.4

**Java**

```
openjdk version "11.0.5" 2019-10-15
```

**PostgreSQL**

13

**Browser**

Firefox 110

**Comment by johnaohara** [comment]
Is this in a local dev environment or in a running prod service? is odd that there is nothing in the server logs

**Comment by whitingjr** [comment]
prod

**Comment by johnaohara** [comment]
Looks like this is being caused by the queries returning empty `values` arrays for the report datasets:

```
2023-02-21 16:07:38,944 4ab0df0ef48d quarkus-run.jar[7] DEBUG [io.hyp.too.hor.svc.ReportServiceImpl] (executor-thread-1014) Dataset 5724/0 (15219) filtered out, value: "6000;2500;4G;Shenandoah"
2023-02-21 16:07:38,947 4ab0df0ef48d quarkus-run.jar[7] DEBUG [io.hyp.too.hor.svc.ReportServiceImpl] (executor-thread-1014) Dataset 5728/0 (15220) filtered out, value: "7500;2500;4G;Shenandoah"
2023-02-21 16:07:38,948 4ab0df0ef48d quarkus-run.jar[7] DEBUG [io.hyp.too.hor.svc.ReportServiceImpl] (executor-thread-1014) Dataset 5731/0 (15221) filtered out, value: "7500;2500;4G;Shenandoah"
2023-02-21 16:07:38,949 4ab0df0ef48d quarkus-run.jar[7] DEBUG [io.hyp.too.hor.svc.ReportServiceImpl] (executor-thread-1014) Table report Quarkus Webprofile GC(303989) includes datasets [15222, 15215, 15210, 15218, 15205, 15223, 15190, 15
189, 15192, 15177, 15174, 15193, 15178, 15211, 15214, 15194, 15184, 15208, 15191, 15224, 15203, 15186, 15183, 15172, 15199, 15176, 15206, 15202, 15200, 15197, 15181, 15225, 15179, 15185, 15216, 15201, 15188, 15175, 15198, 15187, 15173, 1
5226, 15217, 15180, 15209, 15213, 15212, 15195, 15204, 15196, 15182, 15207]
2023-02-21 16:07:38,956 4ab0df0ef48d quarkus-run.jar[7] DEBUG [io.hyp.too.hor.svc.ReportServiceImpl] (executor-thread-1014) Series: {15172="5000;2500;4G;G1", 15173="5000;2500;4G;Throughput", 15174="5000;2500;4G;G1", 15175="1000;2500;4G;S
henandoah", 15176="1000;2500;4G;G1", 15177="1000;2500;4G;ZGC", 15178="1000;2500;4G;Throughput", 15179="2000;2500;4G;Shenandoah", 15180="2000;2500;4G;Shenandoah", 15181="5000;2500;4G;Shenandoah", 15182="5000;2500;4G;ZGC", 15183="2000;2500
;4G;G1", 15184="2000;2500;4G;ZGC", 15185="2000;2500;4G;Throughput", 15186="3000;2500;4G;G1", 15187="3000;2500;4G;ZGC", 15188="3000;2500;4G;Throughput", 15189="4000;2500;4G;G1", 15190="4000;2500;4G;ZGC", 15191="4000;2500;4G;Throughput", 1
5192="3000;2500;4G;Shenandoah", 15193="4000;2500;4G;Shenandoah", 15194="5000;2500;4G;Shenandoah", 15195="1000;2500;4G;Shenandoah", 15196="1000;2500;4G;G1", 15197="1000;2500;4G;ZGC", 15198="1000;2500;4G;Throughput", 15199="2000;2500;4G;Sh
enandoah", 15200="2000;2500;4G;G1", 15201="2000;2500;4G;ZGC", 15202="2000;2500;4G;Throughput", 15203="3000;2500;4G;Shenandoah", 15204="3000;2500;4G;G1", 15205="3000;2500;4G;ZGC", 15206="3000;2500;4G;Throughput", 15207="4000;2500;4G;Shena
ndoah", 15208="4000;2500;4G;G1", 15209="4000;2500;4G;ZGC", 15210="4000;2500;4G;Throughput", 15211="5000;2500;4G;Shenandoah", 15212="5000;2500;4G;G1", 15213="5000;2500;4G;ZGC", 15214="5000;2500;4G;Throughput", 15215="5000;2500;4G;Shenando
ah", 15216="5000;2500;4G;Shenandoah", 15217="5000;2500;4G;Shenandoah", 15218="5000;2500;4G;Shenandoah", 15222="1000;2500;4G;Shenandoah", 15223="2000;2500;4G;Shenandoah", 15224="3000;2500;4G;Shenandoah", 15225="4000;2500;4G;Shenandoah", 1
5226="5000;2500;4G;Shenandoah"}
2023-02-21 16:07:38,963 4ab0df0ef48d quarkus-run.jar[7] DEBUG [io.hyp.too.hor.svc.ReportServiceImpl] (executor-thread-1014) Scales: {15172="5000;2500;4G;G1", 15173="5000;2500;4G;Throughput", 15174="5000;2500;4G;G1", 15175="1000;2500;4G;S
henandoah", 15176="1000;2500;4G;G1", 15177="1000;2500;4G;ZGC", 15178="1000;2500;4G;Throughput", 15179="2000;2500;4G;Shenandoah", 15180="2000;2500;4G;Shenandoah", 15181="5000;2500;4G;Shenandoah", 15182="5000;2500;4G;ZGC", 15183="2000;2500;4G;G1", 15184="2000;2500;4G;ZGC", 15185="2000;2500;4G;Throughput", 15186="3000;2500;4G;G1", 15187="3000;2500;4G;ZGC", 15188="3000;2500;4G;Throughput", 15189="4000;2500;4G;G1", 15190="4000;2500;4G;ZGC", 15191="4000;2500;4G;Throughput", 15192="3000;2500;4G;Shenandoah", 15193="4000;2500;4G;Shenandoah", 15194="5000;2500;4G;Shenandoah", 15195="1000;2500;4G;Shenandoah", 15196="1000;2500;4G;G1", 15197="1000;2500;4G;ZGC", 15198="1000;2500;4G;Throughput", 15199="2000;2500;4G;Shenandoah", 15200="2000;2500;4G;G1", 15201="2000;2500;4G;ZGC", 15202="2000;2500;4G;Throughput", 15203="3000;2500;4G;Shenandoah", 15204="3000;2500;4G;G1", 15205="3000;2500;4G;ZGC", 15206="3000;2500;4G;Throughput", 15207="4000;2500;4G;Shenandoah", 15208="4000;2500;4G;G1", 15209="4000;2500;4G;ZGC", 15210="4000;2500;4G;Throughput", 15211="5000;2500;4G;Shenandoah", 15212="5000;2500;4G;G1", 15213="5000;2500;4G;ZGC", 15214="5000;2500;4G;Throughput", 15215="5000;2500;4G;Shenandoah", 15216="5000;2500;4G;Shenandoah", 15217="5000;2500;4G;Shenandoah", 15218="5000;2500;4G;Shenandoah", 15222="1000;2500;4G;Shenandoah", 15223="2000;2500;4G;Shenandoah", 15224="3000;2500;4G;Shenandoah", 15225="4000;2500;4G;Shenandoah", 15226="5000;2500;4G;Shenandoah"}
2023-02-21 16:07:38,973 4ab0df0ef48d quarkus-run.jar[7] DEBUG [io.hyp.too.hor.svc.ReportServiceImpl] (executor-thread-1014) Data per dataset: {15172=TableReport.Data{datasetId=15172, category='', series='G1', label='5000', values=[]}, 15173=TableReport.Data{datasetId=15173, category='', series='Throughput', label='5000', values=[]}, 15174=TableReport.Data{datasetId=15174, category='', series='G1', label='5000', values=[]}, 15175=TableReport.Data{datasetId=15175, category='', series='Shenandoah', label='1000', values=[]}, 15176=TableReport.Data{datasetId=15176, category='', series='G1', label='1000', values=[]}, 15177=TableReport.Data{datasetId=15177, category='', series='ZGC', label='1000', values=[]}, 15178=TableReport.Data{datasetId=15178, category='', series='Throughput', label='1000', values=[]}, 15179=TableReport.Data{datasetId=15179, category='', series='Shenandoah', label='2000', values=[]}, 15180=TableReport.Data{datasetId=15180, category='', series='Shenandoah', label='2000', values=[]}, 15181=TableReport.Data{datasetId=15181, category='', series='Shenandoah', label='5000', values=[]}, 15182=TableReport.Data{datasetId=15182, category='', series='ZGC', label='5000', values=[]}, 15183=TableReport.Data{datasetId=15183, category='', series='G1', label='2000', values=[]}, 15184=TableReport.Data{datasetId=15184, category='', series='ZGC', label='2000', values=[]}, 15185=TableReport.Data{datasetId=15185, category='', series='Throughput', label='2000', values=[]}, 15186=TableReport.Data{datasetId=15186, category='', series='G1', label='3000', values=[]}, 15187=TableReport.Data{datasetId=15187, category='', series='ZGC', label='3000', values=[]}, 15188=TableReport.Data{datasetId=15188, category='', series='Throughput', label='3000', values=[]}, 15189=TableReport.Data{datasetId=15189, category='', series='G1', label='4000', values=[]}, 15190=TableReport.Data{datasetId=15190, category='', series='ZGC', label='4000', values=[]}, 15191=TableReport.Data{datasetId=15191, category='', series='Throughput', label='4000', values=[]}, 15192=TableReport.Data{datasetId=15192, category='', series='Shenandoah', label='3000', values=[]}, 15193=TableReport.Data{datasetId=15193, category='', series='Shenandoah', label='4000', values=[]}, 15194=TableReport.Data{datasetId=15194, category='', series='Shenandoah', label='5000', values=[]}, 15195=TableReport.Data{datasetId=15195, category='', series='Shenandoah', label='1000', values=[]}, 15196=TableReport.Data{datasetId=15196, category='', series='G1', label='1000', values=[]}, 15197=TableReport.Data{datasetId=15197, category='', series='ZGC', label='1000', values=[]}, 15198=TableReport.Data{datasetId=15198, category='', series='Throughput', label='1000', values=[]}, 15199=TableReport.Data{datasetId=15199, category='', series='Shenandoah', label='2000', values=[]}, 15200=TableReport.Data{datasetId=15200, category='', series='G1', label='2000', values=[]}, 15201=TableReport.Data{datasetId=15201, category='', series='ZGC', label='2000', values=[]}, 15202=TableReport.Data{datasetId=15202, category='', series='Throughput', label='2000', values=[]}, 15203=TableReport.Data{datasetId=15203, category='', series='Shenandoah', label='3000', values=[]}, 15204=TableReport.Data{datasetId=15204, category='', series='G1', label='3000', values=[]}, 15205=TableReport.Data{datasetId=15205, category='', series='ZGC', label='3000', values=[]}, 15206=TableReport.Data{datasetId=15206, category='', series='Throughput', label='3000', values=[]}, 15207=TableReport.Data{datasetId=15207, category='', series='Shenandoah', label='4000', values=[]}, 15208=TableReport.Data{datasetId=15208, category='', series='G1', label='4000', values=[]}, 15209=TableReport.Data{datasetId=15209, category='', series='ZGC', label='4000', values=[]}, 15210=TableReport.Data{datasetId=15210, category='', series='Throughput', label='4000', values=[]}, 15211=TableReport.Data{datasetId=15211, category='', series='Shenandoah', label='5000', values=[]}, 15212=TableReport.Data{datasetId=15212, category='', series='G1', label='5000', values=[]}, 15213=TableReport.Data{datasetId=15213, category='', series='ZGC', label='5000', values=[]}, 15214=TableReport.Data{datasetId=15214, category='', series='Throughput', label='5000', values=[]}, 15215=TableReport.Data{datasetId=15215, category='', series='Shenandoah', label='5000', values=[]}, 15216=TableReport.Data{datasetId=15216, category='', series='Shenandoah', label='5000', values=[]}, 15217=TableReport.Data{datasetId=15217, category='', series='Shenandoah', label='5000', values=[]}, 15218=TableReport.Data{datasetId=15218, category='', series='Shenandoah', label='5000', values=[]}, 15222=TableReport.Data{datasetId=15222, category='', series='Shenandoah', label='1000', values=[]}, 15223=TableReport.Data{datasetId=15223, category='', series='Shenandoah', label='2000', values=[]}, 15224=TableReport.Data{datasetId=15224, category='', series='Shenandoah', label='3000', values=[]}, 15225=TableReport.Data{datasetId=15225, category='', series='Shenandoah', label='4000', values=[]}, 15226=TableReport.Data{datasetId=15226, category='', series='Shenandoah', label='5000', values=[]}}
2023-02-21 16:07:38,985 4ab0df0ef48d quarkus-run.jar[7] DEBUG [io.hyp.too.hor.svc.ReportServiceImpl] (executor-thread-1014) Caused by exception: org.graalvm.polyglot.PolyglotException: TypeError: Cannot read property 'forEach' of undefined
        at <js>.__func(Unnamed:4)
        at <js>.:program(Unnamed:12)
        at org.graalvm.polyglot.Context.eval(Context.java:425)
        at io.hyperfoil.tools.horreum.svc.ReportServiceImpl.lambda$createTableReport$8(ReportServiceImpl.java:405)
        at io.hyperfoil.tools.horreum.svc.ReportServiceImpl.executeInContext(ReportServiceImpl.java:709)
        at io.hyperfoil.tools.horreum.svc.ReportServiceImpl.createTableReport(ReportServiceImpl.java:383)
        at io.hyperfoil.tools.horreum.svc.ReportServiceImpl.previewTableReport(ReportServiceImpl.java:301)
        at io.hyperfoil.tools.horreum.svc.ReportServiceImpl_Subclass.previewTableReport$$superforward1(Unknown Source)
        at io.hyperfoil.tools.horreum.svc.ReportServiceImpl_Subclass$$function$$4.apply(Unknown Source)
        at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:54)
        at io.hyperfoil.tools.horreum.server.RolesInterceptor.intercept(RolesInterceptor.java:70)
        at io.hyperfoil.tools.horreum.server.RolesInterceptor_Bean.intercept(Unknown Source)
        at io.quarkus.arc.impl.InterceptorInvocation.invoke(InterceptorInvocation.java:42)
        at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:50)
        at io.quarkus.security.runtime.interceptor.SecurityHandler.handle(SecurityHandler.java:47)
        at io.quarkus.security.runtime.interceptor.PermitAllInterceptor.intercept(PermitAllInterceptor.java:23)
        at io.quarkus.security.runtime.interceptor.PermitAllInterceptor_Bean.intercept(Unknown Source)
        at io.quarkus.arc.impl.InterceptorInvocation.invoke(InterceptorInvocation.java:42)
        at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:50)
        at io.quarkus.resteasy.reactive.server.runtime.StandardSecurityCheckInterceptor.intercept(StandardSecurityCheckInterceptor.java:43)
        at io.quarkus.resteasy.reactive.server.runtime.StandardSecurityCheckInterceptor_PermitAllInterceptor_Bean.intercept(Unknown Source)
        at io.quarkus.arc.impl.InterceptorInvocation.invoke(InterceptorInvocation.java:42)
        at io.quarkus.arc.impl.AroundInvokeInvocationContext.perform(AroundInvokeInvocationContext.java:41)
        at io.quarkus.arc.impl.InvocationContexts.performAroundInvoke(InvocationContexts.java:33)
        at io.hyperfoil.tools.horreum.svc.ReportServiceImpl_Subclass.previewTableReport(Unknown Source)
        at io.hyperfoil.tools.horreum.svc.ReportServiceImpl_ClientProxy.previewTableReport(Unknown Source)
        at io.hyperfoil.tools.horreum.api.ReportService$quarkusrestinvoker$previewTableReport_a74769530b34789dd7f670099e085c5efdedaa6b.invoke(Unknown Source)
        at org.jboss.resteasy.reactive.server.handlers.InvocationHandler.handle(InvocationHandler.java:29)
        at io.quarkus.resteasy.reactive.server.runtime.QuarkusResteasyReactiveRequestContext.invokeHandler(QuarkusResteasyReactiveRequestContext.java:114)
        at org.jboss.resteasy.reactive.common.core.AbstractResteasyReactiveContext.run(AbstractResteasyReactiveContext.java:145)
        at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:576)
        at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
        at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
        at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:29)
        at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:29)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by whitingjr** [comment]
this is not the report I am using that this issue is created for.

**Comment by whitingjr** [comment]
the issue is due to 
```2023-02-22 16:43:35,839 DEBUG [io.hyp.too.hor.svc.ReportServiceImpl] (executor-thread-1632) Data per dataset: {51633=TableReport.Data{datasetId=51633, category='', series='[65.98,83.96,91.96,79.97,63.98,64.46,63.98,73.97,70.95,75.98,73.98,63.98,65.99,62.99,62.99,63.99,62.99,62.99,62.99,63.99,62.99,62.99,62.99,62.99,63.99,62.99,62.99,76.99,93.98,79.98,72.99,70.99,67.99,67.98,68.99,66.99,67.99,68.99,66.98,66.99,67.99,67.99,67.99,66.99,66.99,67.99,66.99,66.99,66.99,66.99,67.99,67.99,67.99,67.98,66.99,65.99,62.99,62.99,66.99,69.98,70.98,78.98,80.99,81.98,85.98,89.98,78.98,72.99,72.99,72.99,70.99,72.99,71.99,70.99,74.99,72.99,71.99,71.99,71.99,75.98,70.99,71.99,70.99,69.99,70.99,71.99,69.99,69.99,67.99,66.98,68.99,67.99,67.99,65.99,64.99,64.99,63.99,63.98,62.98,62.99,62.99,62.99,62.99,63.98,63.99,62.99,62.99,62.99]', label='5000', values=[]}, 51892=TableReport.Data{datasetId=51892, category='', series='[65.98,82.96,92.96,83.92,63.98,63.98,64.98,73.97,69.98,72.97,72.97,63.98,65.98,62.98,63.99,62.99,62.99,62.99,63.99,62.99,62.99,62.99,63.99,62.99,62.99,62.99,62.99,74.98,88.96,81.97,71.97,71.97,67.98,67.98,68.98,67.97,67.98,67.98,66.98,66.98,67.98,67.98,66.98,66.98,67.98,66.98,67.98,66.98,67.98,66.98,66.98,66.98,66.98,67.98,64.98,62.99,62.99,67.98,68.98,68.98,71.98,75.97,77.97,76.97,79.97,81.97,75.97,72.98,70.98,70.98,69.98,70.97,69.98,69.98,69.97,70.98,70.98,71.98,69.98,71.97,71.98,70.98,73.98,71.98,71.97,72.97,70.98,71.98,66.98,72.98,69.98,68.97,70.98,68.98,70.98,69.97,69.97,70.98,69.98,69.98,69.97,68.98,69.98,67.97,70.97,70.97,69.98,66.98,64.98,64.98,63.98,64.98,64.98,65.98,63.98,64.98,63.98,62.98,63.99,62.99,62.99,62.99,62.99,63.98,63.99,62.99,62.99,62.99]', label='5000', values=[]}, 51622=TableReport.Data{datasetId=51622, category='', series='[{"power":66.97},{"power":85.97},{"power":92.98},{"power":80.98},{"power":63.98},{"power":63.98},{"power":63.98},{"power":75.98},{"power":68.98},{"power":77.98},{"power":71.98},{"power":62.99},{"power":65.99},{"power":62.99},{"power":63.99},{"power":62.99},{"power":62.99},{"power":63.99},{"power":62.99},{"power":62.99},{"power":62.99},{"power":62.99},{"power":63.99},{"power":62.99},{"power":62.99},{"power":62.99},{"power":62.99},{"power":82.98},{"power":90.98},{"power":81.98},{"power":72.98},{"power":71.99},{"power":69.98},{"power":66.99},{"power":68.99},{"power":67.99},{"power":67.99},{"power":68.99},{"power":66.98},{"power":66.99},{"power":68.99},{"power":66.99},{"power":66.99},{"power":67.99},{"power":66.99},{"power":67.99},{"power":66.99},{"power":66.98},{"power":67.99},{"power":67.98},{"power":66.98},{"power":66.98},{"power":67.98},{"power":66.98},{"power":64.99},{"power":63.99},{"power":62.99},{"power":70.99},{"power":76.98},{"power":76.98},{"power":78.99},{"power":81.99},{"power":84.98},{"power":87.98},{"power":79.98},{"power":74.99},{"power":72.99},{"power":73.99},{"power":74.98},{"power":74.98},{"power":75.99},{"power":77.98},{"power":78.98},{"power":76.98},{"power":76.98},{"power":76.98},{"power":76.98},{"power":78.98},{"power":78.98},{"power":77.98},{"power":75.98},{"power":78.98},{"power":75.98},{"power":74.98},{"power":74.98},{"power":72.98},{"power":72.98},{"power":70.98},{"power":69.98},{"power":76.98},{"power":74.98},{"power":73.98},{"power":71.98},{"power":64.99},{"power":63.98},{"power":65.98},{"power":64.98},{"power":63.99},{"power":62.99},{"power":62.99},{"power":62.99},{"power":63.99},{"power":62.98},{"power":63.98},{"power":62.99},{"power":62.99}]', label='5000', values=[]}, 51630=TableReport.Data{datasetId=51630, category='', series='null', label='5000', values=[]}, 51871=TableReport.Data{datasetId=51871, category='', series='[66.97,82.97,92.98,79.99,63.98,63.98,63.98,64.98,64.98,64.98,65.98,63.98,63.98,62.98,62.98,63.99,62.99,62.99,62.99,63.99,62.99,62.99,62.99,62.99,63.99,62.99,62.99,73.99,92.98,81.98,70.99,72.98,68.98,68.99,68.99,67.99,67.99,67.99,66.99,66.99,67.99,68.99,66.99,67.99,66.99,67.99,66.99,66.99,66.99,66.99,67.99,67.99,66.99,66.99,65.99,64.99,62.99,63.99,70.99,70.98,75.98,81.98,82.99,84.98,89.98,84.98,76.99,71.99,70.98,70.99,72.99,73.99,70.99,73.99,71.99,72.99,70.99,71.99,72.99,70.99,72.99,70.98,70.99,70.98,72.98,70.98,69.98,69.98,65.98,67.98,69.98,67.99,66.99,66.99,63.99,63.99,63.98,63.99,62.99,62.99,62.99,62.99,62.98,63.98,62.99,62.99]', label='5000', values=[]}}
2023-02-22 16:43:35,842 ERROR [org.jbo.res.rea.com.cor.AbstractResteasyReactiveContext] (executor-thread-1632) Request failed: com.fasterxml.jackson.databind.JsonMappingException: (was java.lang.NullPointerException) (through reference chain: io.hyperfoil.tools.horreum.entity.report.TableReport["logs"]->java.util.ArrayList[0]->io.hyperfoil.tools.horreum.entity.report.ReportLog["reportId"])
        at com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:402)
        at com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:361)
        at com.fasterxml.jackson.databind.ser.std.StdSerializer.wrapAndThrow(StdSerializer.java:316)
        at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:782)
        at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:178)
        at com.fasterxml.jackson.databind.ser.impl.IndexedListSerializer.serializeContents(IndexedListSerializer.java:119)
        at com.fasterxml.jackson.databind.ser.impl.IndexedListSerializer.serialize(IndexedListSerializer.java:79)
        at com.fasterxml.jackson.databind.ser.impl.IndexedListSerializer.serialize(IndexedListSerializer.java:18)
        at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:733)
        at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:774)
        at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:178)
        at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider._serialize(DefaultSerializerProvider.java:480)
        at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider.serializeValue(DefaultSerializerProvider.java:319)
        at com.fasterxml.jackson.databind.ObjectWriter$Prefetch.serialize(ObjectWriter.java:1572)
        at com.fasterxml.jackson.databind.ObjectWriter._writeValueAndClose(ObjectWriter.java:1273)
        at com.fasterxml.jackson.databind.ObjectWriter.writeValue(ObjectWriter.java:1098)
        at io.quarkus.resteasy.reactive.jackson.runtime.serialisers.BasicServerJacksonMessageBodyWriter.writeResponse(BasicServerJacksonMessageBodyWriter.java:39)
        at org.jboss.resteasy.reactive.server.core.ServerSerialisers.invokeWriter(ServerSerialisers.java:220)
        at org.jboss.resteasy.reactive.server.core.ServerSerialisers.invokeWriter(ServerSerialisers.java:191)
        at org.jboss.resteasy.reactive.server.core.serialization.FixedEntityWriter.write(FixedEntityWriter.java:28)
        at org.jboss.resteasy.reactive.server.handlers.ResponseWriterHandler.handle(ResponseWriterHandler.java:34)
        at io.quarkus.resteasy.reactive.server.runtime.QuarkusResteasyReactiveRequestContext.invokeHandler(QuarkusResteasyReactiveRequestContext.java:120)
        at org.jboss.resteasy.reactive.common.core.AbstractResteasyReactiveContext.run(AbstractResteasyReactiveContext.java:145)
        at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:576)
        at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
        at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
        at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:29)
        at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:29)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.NullPointerException
        at io.hyperfoil.tools.horreum.entity.report.ReportLog.getReportId(ReportLog.java:30)
        at jdk.internal.reflect.GeneratedMethodAccessor226.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:689)
        at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:774)
        ... 26 more

````

**Comment by whitingjr** [comment]
fixed in #432
